src/main/java/com/kulik/airbnb/service/AuthService.java

```diff
@@ -62,19 +65,31 @@ public String loginOrRegisterViaGoogle(String authorizationCode) throws IOExcept
         int status = 0;
         if (userFromDatabase == null) {
             status = userDao.create(userFromGoogle);
+            userDao.confirm(userFromGoogle.getEmail());
         } else {
             status = 1;
         }
 
         if (status > 0) {
             SecurityContextHolder.getContext().setAuthentication(
                     new UsernamePasswordAuthenticationToken(userFromGoogle.getEmail(), ""));
-            String token = jwtTokenProvider.createToken(userFromGoogle.getEmail());
+            String token = jwtTokenProvider.createAuthorizationToken(userFromGoogle.getEmail());
 
             return token;
         } else {
             return null;
         }
     }
 
+    String sendConfirmationLink(String email) throws UnirestException {
+        String confirmationLink = "http://localhost:8080/users/confirmation?token=" + jwtTokenProvider.createConfirmationToken(email);
```

Comment:
tha base URL should be configurable because you won't change this line when deploy it